libraries/ml5.js.yaml

```diff
@@ -0,0 +1,16 @@
+name: ml5.js
+category: ai-ml-cv
+description: Friendlly machine learning for the web—a neighborly approach to creating and exploring artificial intelligence in the browser.
+author:
+  - name: NYU's Interactive Telecommunications/Interactive Media Arts Program
+    url: https://tisch.nyu.edu/itp
+  - name: NYU Shanghai's Interactive Media Arts Program
+    url: https://ima.shanghai.nyu.edu/
+sourceUrl: https://github.com/ml5js/ml5-library
+websiteUrl: https://ml5js.org/
+npm: 'ml5'
+npmFilePath: 'ml5@0.12.2/dist/ml5.min.js'
```

Comment:
For `npmFilePath`, I used 'ml5@0.12.2/dist/ml5.min.js' because https://cdn.jsdelivr.net/npm/ml5@0.12.2/dist/ml5.min.js worked but `dist/ml5.min.js`didn't! Not sure about the `npm` field—@davepagurek please advise :-)

Thanks for reviewing the file Gottfried!

Reply:
Just `dist/ml5.min.js` will do!

Reply:
Updated the file accordingly—thank you both @davepagurek @gohai !

Reply:
Thanks!